app/views/users/_form.html.erb

```diff
@@ -20,7 +20,9 @@
 
   <div class='tab-content'>
     <div class='tab-pane active' id='primary'>
-      <%  msg =  @user.internal? ? "": "External user login can not be changed" %>
+      <% if !(@user.internal?) && !(@user.new_record? ) %>
```

Comment:
The parenthesis are not needed here (or at least the spacing should be consistent